src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -951,20 +951,37 @@ def do_request(params)
   describe 'DELETE #trigger_rebuild' do
     before do
       login(user)
-      delete :trigger_rebuild, params: { project: source_project, package: source_package }
     end
 
-    it 'lets the user know there was an error' do
-      expect(flash[:error]).to_not be_empty
+    context 'when triggering a rebuild fails' do
+      before do
+        delete :trigger_rebuild, params: { project: source_project, package: source_package }
+      end
+
+      it 'lets the user know there was an error' do
+        expect(flash[:error]).to_not be_empty
```

Comment:
Welcome in the 'it makes sense to test error messages' camp:-) I am not alone anymore.

I just touched the line when wrapping the code in a context block. But I would be okay to change the error check